components/script/dom/webidls/DOMQuad.webidl

```diff
@@ -0,0 +1,34 @@
+/* -*- Mode: IDL; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+/*
+ * The origin of this IDL file is
+ * https://drafts.fxtf.org/geometry/#DOMQuad
+ *
+ * Copyright:
+ * To the extent possible under law, the editors have waived all copyright and
+ * related or neighboring rights to this work.
+ */
+
+[Constructor(optional DOMPointInit p1, optional DOMPointInit p2,
+             optional DOMPointInit p3, optional DOMPointInit p4),
+ Exposed=Window,Exposed=Worker]
```

Comment:
This is not the exposed set you are looking for, it should be "Exposed=(Window,Worker)".